src/sentry/quotas/redis.py

```diff
@@ -87,6 +87,22 @@ def get_quotas(self, project, key=None, keys=None):
                     )
                 )
 
+        with sentry_sdk.start_span(op="redis.get_quotas.get_monitor_quota") as span:
+            span.set_tag("project.id", project.id)
+            mrlquota = self.get_monitor_quota(project)
```

Comment:
worth separating into variables instead of unnamed tuple?

Reply:
I'm just following what we're doing above